core/src/core-api/featuretoggles.js

```diff
@@ -54,7 +46,37 @@ class LuigiFeatureToggles {
    * @example Luigi.featureToggles().getActiveFeatureToggleList();
    */
   getActiveFeatureToggleList() {
+    return [...get(this.featureToggleList)].filter(ft => !ft.startsWith('!'));
+  }
+
+  /**
+   * Get a list of active feature toggles
+   * @memberof FeatureToggles
+   * @return {Array} of feature toggles
+   * @example Luigi.featureToggles().getFeatureToggleList();
+   */
+  getFeatureToggleList() {
     return [...get(this.featureToggleList)];
```

Comment:
🤔 I am not sure if we need this function anymore, but for the time being seems like its not used anywhere. Unless this was specifically requested for something planned in the future, I would remove altogether.